pandas/_libs/tslibs/conversion.pyx

```diff
@@ -608,32 +608,33 @@ cpdef inline datetime localize_pydatetime(datetime dt, object tz):
 # ----------------------------------------------------------------------
 # Timezone Conversion
 
-cdef inline int64_t tz_convert_tzlocal_to_utc(int64_t val, tzinfo tz):
+cdef inline int64_t _tz_convert_tzlocal_utc(int64_t val, tzinfo tz,
+                                            bint to_utc=True):
```

Comment:
looks good. can you add what this function does in doc-string. and mark as this is private.

Reply:
Mind if I put the docsting in the follow-up?  There’s a couple more coming up in the next few hours.

Reply:
sure